frontend/src/models/behaviors.js

```diff
@@ -26,3 +27,111 @@ export class RandomCardBehavior {
     return null;
   }
 }
+
+export class BasicRuleBasedBehaviour {
+  announcementToMake(possibleAnnouncements) {
+    // ToDo announce by "goodies"
+    /**
+     * for instance
+     * only one fehlsuit
+     * high number of trumps
+     * high trumps, such as 2x ten of hearts
+     * blank ace and starting
+     */
+    return null;
+  }
+
+  cardToPlay(hand, trick, memory) {
+    let baseCard = trick.baseCard();
+    if (!baseCard) {
+      /** It's our turn. Decide how to deal with cards */
+      return this.startingRule(hand, memory);
+    }
+    if (!baseCard.isTrump()) {
+      /** We need to serve a fehl card */
+      return this.fehlRule(hand, trick, memory);
+    }
+    // ToDo how to play if not starting or mustn't serve fehl
+    // i'm thinking of something working with expectation value
+    return sample(playableCards(hand.cards, baseCard));
+  }
+
+  startingRule(hand, memory) {
+    for (const suit of [suits.clubs, suits.spades, suits.hearts]) {
+      if (hand.hasBlankAce(suit) && !memory.fehlSuitPlayedBefore(suit)) {
+        return hand.fehlCardsBySuit(suit)[0];
+      }
+    }
+    // ToDo check if we know with whom we play and if we want to play a strategy
+    return sample(playableCards(hand.cards, undefined));
+  }
+
+  fehlRule(hand, trick, memory) {
+    let baseCard = trick.baseCard();
+    if (hand.mustServeSuit(baseCard.suit)) {
+      let fehlCards = hand.fehlCardsBySuit(baseCard.suit);
+      let highest = fehlCards[0];
+      let lowest = fehlCards.reverse()[0];
```

Comment:
Hah!
You got me on that one. Although I believe we probably won't run into an issue here, this is definately error prone and a very good thing to avoid!